presentation/components/src/presentation-components/common/ContentBuilder.ts

```diff
@@ -303,30 +290,66 @@ function convertNestedContentFieldHierarchyToStructArrayHierarchy(fieldHierarchy
   return convertedFieldHierarchy;
 }
 
-function createPropertyRecordPropsFromFieldInfo(fieldType: TypeDescription, name: string, label: string, isReadonly?: boolean): CreatePropertyRecordProps {
+interface FieldInfo {
+  type: TypeDescription;
+  name: string;
+  label: string;
+  renderer?: RendererDescription;
+  editor?: EditorDescription;
+  enum?: EnumerationInfo;
+  isReadonly?: boolean;
+}
+function createPropertyDescriptionFromFieldInfo(info: FieldInfo) {

```

Comment:
Please specify return type

Reply:
Why do you think it's necessary? IMO, it's not:
- It's a non-exported function, so will only be used from within this module where content of the function is seen. The return type can be inferred from the name of the function and the type of object it returns (see below).
- Typescript can infer the type from the type of object the function returns (clearly defined as `PropertyDescription`)

Reply:
It would be nice to know function's return type without requiring the reader to search for the return statement or pulling this PR, opening this file in vscode, and hovering over the function name.

Aside from that, having function's return type explicitly stated prevents future changes from inadvertently changing it to something unexpected.